### PR TITLE
Tag import error codes

### PR DESCRIPTION
**Pull request by lbosque**
Depending if there is a user expected error or a platform unexpected one.

In order to do that I've tagged every error with 'user' or 'cartodb'.

Please CR @rafatower 

Also, @javisantana @saleiva please review the set tags per code

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3550/))

File: lib/cartodb/import_error_codes.rb [hunk 1]
```diff
@@ -1,167 +1,212 @@
...
     1001 => {
       title: 'Download error',
-      what_about: "The remote URL returned an error. Please verify your file is available at that URL."
+      what_about: "The remote URL returned an error. Please verify your file is available at that URL.",
+      source: ERROR_SOURCE_USER
     },
     1002 => {
       title: 'Unsupported/Unrecognized file type',
-      what_about: "Should we support this filetype? Let us know in our <a href='mailto:support@cartodb.com'>support email</a>!"
+      what_about: "Should we support this filetype? Let us know in our <a href='mailto:support@cartodb.com'>support email</a>!",
```
Comment (rafatower) [inline lib/cartodb/import_error_codes.rb]: @Kartones @juanignaciosl that's the kind of change why I prefer to keep the trailing comma in languages that allow it

**Comment by rafatower** [comment]
:+1:

File: lib/cartodb/import_error_codes.rb (hunk 1 above)
Comment (Kartones) [inline lib/cartodb/import_error_codes.rb]: Trailing comma is for cowards and bad developers!

I dislike trailing commas a lot, but if everyone agrees I'd try to stick to them. Else I'll keep not adding them

Comment (juanignaciosl) [inline lib/cartodb/import_error_codes.rb]: I don't like them either. [Guide tells not to use them indeed](https://github.com/Vizzuality/ruby-style-guide).

**Comment by lbosque** [comment]
Deploying it. We can switch tags for a error code if we find necessary at any moment.

**Pull request merged**